src/resolve/resolved-dns-transaction.h

```diff
@@ -61,6 +62,8 @@ struct DnsTransaction {
 
         DnsAnswer *answer;
         int answer_rcode;
+        int answer_ede_rcode;
+        char *answer_ede_msg;
```

Comment:
this field is only initialized in case of dnssec right?

that would also suggest it should carry "dnssec" in the name somehow.

that said, if we have an EDE error, maybe we should always propagate it back here? (and potentially back to clients, at least if they come via easily extensible varlink or so)

[but don't want to put more work on you here, we can do that some other time]

Reply:
Yes. I agree, I think it should probably always be propagated. I also think the other t->answer* fields probably should as well, especially now that most decisions follow the dns_extract_answer call. Those touch ups can probably be considered in a follow-up pr.